etcdmain/config.go

```diff
@@ -283,6 +283,11 @@ func (cfg *config) parse(arguments []string) error {
 	}
 
 	var err error
+
+	if cfg.configFile == "" {
+		cfg.configFile = os.Getenv("ETCD_CONFIG_FILE")
```

Comment:
Instead of hard-coding this env name, use `FlagToEnv()`.
https://github.com/etcd-io/etcd/blob/e8b940f268a80c14f7082589f60cbfd3de531d12/pkg/flags/flag.go#L70